Public/Src/Tools/ContentPlacementAnalysisTools/ContentPlacementAnalysisTools.Extraction/ContentPlacementAnalysisTools.Extraction.dsc

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Tools.ContentPlacement.Extraction {
+
+    export declare const qualifier: BuildXLSdk.FullFrameworkQualifier;
+
+    // this is the log file config for cptools.builddownloader.exe
+    @@public
+    export const buildDownloaderLogFile = f`cptools.builddownloader.exe.nlog`;
+
+    // these are the static resources i use
+    @@public
+    export const staticResources = {

```

Comment:
>staticResources [](start = 17, length = 15)

if this layout is needed by the cptools.builddownloader then it would make sense to just add it to the executable
using the `runtimeContent` field like:

```
export const exe = BuildXLSdk.executable({
        assemblyName: "cptools.builddownloader",
        ...
       runtimeContent: [
            staticResources
       ],
});
```

similar to:
\Public\Src\Cache\MemoizationStore\Library\BuildXL.Cache.MemoizationStore.Library.dsc